src/metabase/models/table.clj

```diff
@@ -311,7 +311,7 @@
                   :view-count    true
                   :created-at    true
                   :updated-at    true}
-   :search-terms [:name :description :display_name]
+   :search-terms [:name :display_name :description]
```

Comment:
Reordered so primary terms come first. Foreshadowing the primary / secondary spec split.